Workbooks/WorkbooksMetadata.json

```diff
@@ -1429,5 +1429,18 @@
       "subtitle": "",
       "provider": "Microsoft",
       "featureFlag": "AzurePurview"
-    }   
+    },
+    {
+      "workbookKey": "ADXvsLA",
+      "logoFileName": "Azure_Sentinel.svg",
+      "description": "This workbook shows the tables from Azure Sentinel which are backed up in ADX. It also provides a comparison between the entries in the Azure Sentinel tables and the ADX tables. Lastly some general information about the queries and ingestion on ADX is shown.",
+      "dataTypesDependencies": [],
```

Comment:
Just wanted to confirm, you don't want any Data connector and Data Type to be linked with this workbook?

Reply:
No since the data connector depends on what you send to ADX and what you have in your environment
So it could be a lot of different data connectors or just one (depending on adx and LA environment)